manager/controllers/app/fybrik_notebook_read_flow_test.go

```diff
@@ -117,15 +118,108 @@ func TestS3NotebookReadFlow(t *testing.T) {
 	// Module installed by setup script directly from remote arrow-flight-module repository
 	// Installing application
 	application := &fapp.FybrikApplication{}
-	g.Expect(readObjectFromFile("../../testdata/notebook/read-flow/fybrikapplication.yaml", application)).ToNot(gomega.HaveOccurred())
+	plotter := &fapp.Plotter{}
+	var applicationKey client.ObjectKey
+	var plotterObjectKey client.ObjectKey
+	var modulesNamespace string
+
+	// Check allow-by-default in case the values is notebook-test-readflow.values.yaml
+	if valuesYaml == "charts/fybrik/notebook-test-readflow.values.yaml" {
+		// Starting allow-by-default
+		fmt.Println("Starting allow-by-default read")
+
+		// Module installed by setup script directly from remote arrow-flight-module repository
+		// Installing application
+
+		g.Expect(readObjectFromFile("../../testdata/notebook/read-flow/fybrikapplication.yaml", application)).
+			ToNot(gomega.HaveOccurred())
+		application.ObjectMeta.Name += "-1"
+		application.Spec.Data[0].DataSetID = catalogedAsset
+		applicationKey = client.ObjectKeyFromObject(application)
+
+		// Create FybrikApplication and FybrikModule
+		fmt.Println("Expecting application creation to succeed")
+		g.Expect(k8sClient.Create(context.Background(), application)).Should(gomega.Succeed())
+
+		fmt.Println("Expecting application to be created")
+		g.Eventually(func() error {
+			return k8sClient.Get(context.Background(), applicationKey, application)
+		}, timeout, interval).Should(gomega.Succeed())
+		fmt.Println("Expecting plotter to be constructed")
+		g.Eventually(func() *fapp.ResourceReference {
+			_ = k8sClient.Get(context.Background(), applicationKey, application)
+			return application.Status.Generated
+		}, timeout, interval).ShouldNot(gomega.BeNil())
+
+		// The plotter has to be created
+
+		plotterObjectKey = client.ObjectKey{Namespace: application.Status.Generated.Namespace,
+			Name: application.Status.Generated.Name}
+		fmt.Println("Expecting plotter to be fetchable")
+		g.Eventually(func() error {
+			return k8sClient.Get(context.Background(), plotterObjectKey, plotter)
+		}, timeout, interval).Should(gomega.Succeed())
+
+		fmt.Println("Expecting application to be ready")
+		g.Eventually(func() bool {
+			err = k8sClient.Get(context.Background(), applicationKey, application)
+			if err != nil {
+				return false
+			}
+			return application.Status.Ready
+		}, timeout, interval).Should(gomega.Equal(true))
+
+		modulesNamespace = plotter.Spec.ModulesNamespace
+		fmt.Printf("data access module namespace notebook test: %s\n", modulesNamespace)
+		g.Expect(application.Status.AssetStates[catalogedAsset].Conditions[ReadyConditionIndex].Status).To(gomega.Equal(v1.ConditionTrue))
+		g.Expect(application.Status.AssetStates[catalogedAsset].Endpoint.Name).ToNot(gomega.BeEmpty())
```

Comment:
indentation?

Reply:
Which line is problematic?

Reply:
ok, it appears as a very long line: 178, also - 189